packages/clarity-js/src/core/version.ts

```diff
@@ -1,2 +1,2 @@
-let version = "0.6.42";
+let version = "0.6.43";
```

Comment:
Please take a look at other PRs; there are few more files where you will need to update the version - including decode, visualize, etc.